src/main/java/ch/njol/skript/conditions/CondIsScreaming.java

```diff
@@ -0,0 +1,44 @@
+package ch.njol.skript.conditions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.conditions.base.PropertyCondition;
+import ch.njol.skript.doc.*;
+import org.bukkit.entity.Enderman;
+import org.bukkit.entity.Goat;
+import org.bukkit.entity.LivingEntity;
+
+@Name("Is Screaming")
+@Description("Check if a goat or enderman (Paper) is or is not screaming.")
```

Comment:
This is just me being picky, but I feel like this could be shortened.
(Also I feel like the "Paper" part isn't needed, since you have it in required plugins)
```suggestion
@Description("Check whether a goat or enderman is screaming.")
```